Themis/Views/Assessment/Modeling Exercise/Diagram Renderers/UMLDiagramRenderer.swift

```diff
@@ -8,5 +8,8 @@
 import Foundation
 
 protocol UMLDiagramRenderer {
+    /// This instance should be used for all actions that one would normally perform with a `GraphicsContext`
+    var context: UMLGraphicsContext { get set }
+
```

Comment:
Should we add font size and canvasBounds to the protocol? What do you think?

Reply:
All conforming structs need those fields. So I think it is a good idea 👍